app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,36 @@
+class CommentsController < ApplicationController
+  def show
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.find(params[:id])
+  end
+
+  def edit
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.find(params[:id])
+  end
+
+  def update
+    @post = Post.find(params[:post_id])
+    @comment= @post.comments.find(params[:id])
+    @comment.update(comment_params)
+    redirect_to edit_post_comment_path, notice: "Comment Successfully Updated"
+  end
```

Comment:
You could also have added some error handling here like you did in your Posts controller